run.py

```diff
@@ -38,23 +38,38 @@ def returnBal(username):
 	pending_balance = data['data']['balances'][0]['pending_received_balance']
 	return (balance, pending_balance)
 
-def process(message,username,chatid):
+def process(message,firstname,username,chatid):
 	message = message.split(" ")
 	for i in range(message.count(' ')):
 		message.remove(' ')
 
-	if "/register" in message[0]:
+# /start
+	if "/start" in message[0]:
+		try:
+			sendMsg(str(firstname)+" Welcome To The PeakShift Tip Bot\n\nhere's how this bot works.\n\nYou can use @dogeshift_bot by messaging it directly or in a group that it is a part of.\n\nAvailable Commands\n\n/register - Register Your Self In the bot\n/tip @username 10 doge - use this To Tip some doge from your balance to another user\n/address - Get Your Deposit Address\n/withdraw 100 <address> - to withdraw Your balance",chatid)
```

Comment:
```suggestion
			sendMsg(str(firstname)+" welcome. I'm the the Peak Shift @ Work Bot\n\nHere's how it works.\n\nYou can use @dogeshift_bot by messaging it directly or in a group that it is a part of.\n\nAvailable Commands\n\n/register - Registers your username with the bot\n/tip @username 10 doge - use this to tip some doge from your balance to another user\n/address - Get your deposit address\n/withdraw 100 <address> - to withdraw your balance",chatid)
```

Reply:
For start command we can leave it with first-name and in others the username.

There isn't much difference between them but when a user clicks on bot username the first come he gets is /start so he must get some respons on start !